webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -0,0 +1,94 @@
+import {
+  changeOrganization,
+  changeLocation,
+  navigateTo,
+} from '../../../foreman_navigation';
+
+export const getCurrentPath = () => window.location.pathname;
+
+export const combineMenuItems = (data) => {
+  const items = [];
+
+  data.menu.forEach((item) => {
+    items.push(item);
+  });
+
+  if (data.taxonomies.organizations) {
+    items.push(createOrgItem(data.organizations.available_organizations));
+  }
+
+  if (data.taxonomies.locations) {
+    items.push(createLocationItem(data.locations.available_locations));
+  }
+  return items;
+};
+
+const createOrgItem = (orgs) => {
+  const anyOrg = {
+    name: 'Any Organization',
+    onClick: () => {
+      changeOrganization('Any Organization');
+      navigateTo('/organizations/clear');
+    },
+  };
+  const childrenArray = [];
+  childrenArray.push(anyOrg);
+
+  orgs.forEach((org) => {
+    const childObject = {
+      type: org.type,
+      name: org.title,
+      onClick: () => {
+        navigateTo(org.href);
```

Comment:
just wondering, does this create a function for each org, or reuse the same function pointer?

Reply:
Its for mobile users that switched orgs. so a function for each org yes.